skyVegasTests/utilities/validateConsoleMessagesHelper.ts

```diff
@@ -1,22 +1,18 @@
-import { test, expect, type Page } from "@playwright/test";
-import { error } from "console";
-import * as fs from "fs";
+import { expect } from "@playwright/test";
+export const messageExists = (consoleMessages: string[], expectedMessage: string):boolean => consoleMessages.some((msg) => msg.includes(expectedMessage));
 
-export let validateConsoleMessages = async (
-  page: Page,
+export const validateConsoleMessages = async (
   expectedMessage: string,
   consoleMessages: string[]
 ) => {
   if (expectedMessage === undefined){
     throw new Error("expected message is null")
   }
   await expect(async () => {
-    const messageExists = consoleMessages.some((msg) =>
-      msg.includes(expectedMessage)
-    );
-    expect(messageExists).toBe(true);
+    expect(messageExists(consoleMessages, expectedMessage)).toBe(true);
   }).toPass({
-    intervals: [5_000, 10_000, 15_000],
-    timeout: 60_000,
+    intervals: [5_000, 10_000, 15_000, 60_000, 70_000, 75_000, 85_000, 95_000],
+    timeout: 120_000,
```

Comment:
Any reason for waiting 2 mins?

Reply:
I think it needs to be 2 mins as we need to have time to make another spin to trigger the reality check message and for the message to appear in the console.

Reply:
We are going to make a few changes so that this test has its own timeout

Reply:
We have added optional parameters, and customized the timeout and intervals for this test. The other tests have a default of 1 minute timeout now :)